src/test/java/com/exasol/adapter/dialects/db2/IntegrationTestConfiguration.java

```diff
@@ -0,0 +1,19 @@
+package com.exasol.adapter.dialects.db2;
+
+import java.nio.file.Path;
+
+public class IntegrationTestConfiguration {
+    public static final String EXASOL_DOCKER_REFERENCE = "7.0.7";
+    public static final String EDB2_DOCKER_REFERENCE = "ibmcom/db2:11.5.5.0";
+    public static final String VIRTUAL_SCHEMAS_JAR_NAME_AND_VERSION = "virtual-schema-dist-9.0.1-db2-2.0.0.jar";
+    public static final Path PATH_TO_VIRTUAL_SCHEMAS_JAR = Path.of("target", VIRTUAL_SCHEMAS_JAR_NAME_AND_VERSION);
+    public static final String DOCKER_IP_ADDRESS = "172.17.0.1";
```

Comment:
that IP address might be different on different systems.

Reply:
This is how it's currently work in all VS integration tests. Do you know how to connect to containers without using this ip?

Reply:
the trick is to go over the host. You can use `getHostIp()` on the container to get the IP address of the host from within the container.
In addition, you need to expose the ports to the host-

Reply:
AFAIR the `getHostIP()` method is quite unstable.

Reply:
well it might not work on windows, but the current solution just works by accident...

Reply:
Current solution is the default address of the docker bridge network. It's also unstable, because it's possible to change it, but usually nobody does it. So I'm not sure which solution is better

Reply:
In case something can be improved we can now improve it in one place...